src/Controller/OpenyMemberships.php

```diff
@@ -351,13 +351,15 @@ public function getProductsInBranch($branch, $data) {
             'variations' => [],
           ];
           foreach ($product->variations as $variant) {
-            $products[$product->uuid()]['variations'][] = [
-              'uuid' => $variant->entity->uuid(),
-              'id' => $variant->entity->id(),
-              'price' => $variant->entity->getPrice()->toArray()['number'],
-              'field_best_value' => $variant->entity->field_best_value->value,
-              'title' => $variant->entity->label(),
-            ];
+            if ($variant->entity) {
```

Comment:
Hi, @froboy .
Lets try to check it better
```suggestion
            if ($variant->entity instance of ProductVariationInterface) {
```